File: python/sum_of_prime_numbers.py
# Identify the prime numbers within the array, then add all of those prime numbers

def is_prime(number):
  if number == 0 or number == 1:
    return False

  for i in range(2, number - 1):
    if (number % i) == 0:
      return False
  return True

def sum_of_prime(arr):
  just_prime = []
  for num in arr:
    if is_prime(num):
      just_prime.append(num)
 
  print(sum(just_prime))

sum_of_prime([1, 23, 31, 50, 53, 76, 99])

File: ruby/sum_of_prime_numbers.rb
# Identify the prime numbers within the array, then add all of those prime numbers

def is_prime?(number)
  if number == 0 || number == 1
    return false
  end

  for i in (2..number-1) do
    if (number % i) == 0
      return false
    end
  end
  true
end

def sum_of_prime(arr)
  just_prime = []
  arr.each do |num|
    if is_prime?(num) 
      just_prime << num
    end
  end
  
  puts just_prime.sum
end

sum_of_prime([1, 23, 31, 50, 53, 76, 99])
